### PR TITLE
atactk recipe

### PR DESCRIPTION
**Pull request by egeogretmen**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

The test for the installed script called "trim_adapters" is "trim_adapters --help" which returns exit code 0 when installed, however it fails during CircleCI builds.

```
+ trim_adapters --help
/opt/conda/conda-bld/atactk_1528493079303/test_tmp/run_test.sh: /opt/conda/conda-bld/atactk_1528493079303/_t_env/bin/trim_adapters: /opt/conda/conda-bld/atactk_1528493079303/_t_env/bin/python: bad interpreter: No such file or directory
```

The shebang is `#!/usr/bin/env python` so it shouldn't be causing a problem. Does anybody have an idea what the problem might be?

### CODE REVIEW

File: recipes/atactk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - pysam
+        - python-levenshtein
+        - sexpdata
```
Comment (bgruening) [inline recipes/atactk/meta.yaml]: runtime dependencies are missing

File: recipes/atactk/build.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+pip install .
```
Comment (bgruening) [inline recipes/atactk/build.sh]: Use that here: https://github.com/bioconda/bioconda-recipes/blob/eaf59a144191c3989ee0c2bd0d799fb0a9281f3e/recipes/bracken/build.sh#L5 for your scripts.

File: recipes/atactk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+test:
+    commands:
+        - trim_adapters --help
+        - make_cut_matrix --help
+
+extra:
+    recipe-maintainers: 
+        - egeogretmen
+    notes:
+        - Adds 3 scripts, namely "trim_adapters", "make_cut_matrix" and "plot_aggregate_matrix.R"
```
Comment (bgruening) [inline recipes/atactk/meta.yaml]: test all 3 scripts

Comment (egeogretmen) [reply recipes/atactk/meta.yaml]: Is there a way to test Rscripts easily? They have to be explicitly coded to print a help text I guess. How can I test [this](https://github.com/ParkerLab/atactk/blob/master/scripts/plot_aggregate_matrix.R) Rscript?

Comment (bgruening) [reply recipes/atactk/meta.yaml]: Just import the R modules: RColorBrewer, ggplot2, grid, gtools. That should be enough, you also need to depend on this as runtime dependency.

See here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/r-gbm/meta.yaml#L37

**Comment by jerowe** [comment]
Thanks!

**Pull request merged**